cloud-functions/src/utils/webHook.ts

```
@@ -0,0 +1,81 @@
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+interface IGenericObject {
+	[key: string]: any;
+}
+
+interface IPostEdit {
+	indexOrHash: string;
+	proposalType: string;
+	title: string;
+	content: string;
+	authorId: Number;
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)